contracts/stellar-axelar-operators/src/tests/test.rs

```diff
@@ -185,3 +185,16 @@ fn execute_fails_when_target_returns_error() {
         )
     );
 }
+
+#[test]
+fn migrate_succeeds() {
+    let (env, client, _) = setup_env();
+
+    let operator = Address::generate(&env);
+
+    client.add_operator(&operator);
```

Comment:
This uses the new storage type, so it's not actually testing the migration. You can test migration via this style, set the old value and then call migrate and check
https://github.com/axelarnetwork/axelar-cgp-stellar/blob/main/packages/stellar-axelar-std/src/interfaces/upgradable.rs#L244